### PR TITLE
install_dir_gir and install_dir_typelib aren't deprecated

### PR DESCRIPTION
**Pull request by ptomato**
Although it's listed kind of unclearly in the Meson docs, these keyword arguments are actually not deprecated. It's only passing a boolean to them that is deprecated, though.

~~I'm not sure how to indicate that booleans used to be allowed but are now deprecated. So I've just removed the boolean type from the type definition.~~

### CODE REVIEW

File: src/libtypenamespace/methods.def [hunk 1]
```diff
@@ -2014,12 +2014,10 @@ gnome_module::generate_gir:
       - Optional: true
       - Types:
         - str
-        - bool
```
Comment (JCWasmx86) [inline src/libtypenamespace/methods.def]: Could you please revert the changes to this file? Otherwise people passing a bool to it would get an error

**Review by JCWasmx86 — COMMENTED** [review]
Hey, thanks for your PR. Sadly these type/function definitions etc. are not flexible enough to tell that just some subset of types is deprecated, so I can't emit a warning for that for now.

The CI failures are unrelated

File: src/libtypenamespace/methods.def (hunk 1 above)
Comment (ptomato) [reply src/libtypenamespace/methods.def]: Sure, makes sense. Done.

**Comment by JCWasmx86** [comment]
Ok, thanks :)
